README.rst

```diff
@@ -87,6 +87,7 @@ Usage
 
   $ pytest --picked --mode=unstaged  # default
 
+  $ pytest --picked --mode=onlychanged
 
 Features
```

Comment:
Would be nice to add a line here about this feature too.

Reply:
done !